From f4f0629e1b11b904cc677ead2f527bcda48cf915 Mon Sep 17 00:00:00 2001
From: Ashok Kumar <ashok2.kumar@amd.com>
Date: Tue, 31 Jan 2023 12:57:33 -0600
Subject: [PATCH] Power-cap -CPUID1 and CPUID2 will not be used for Processor
 Info

CPUID1 and CPUID2 will not be used for Processor Information,
instead /system/processor will be used.

Signed-off-by: Ashok Kumar <ashok2.kumar@amd.com>
---
 xyz/openbmc_project/Control/Power/Cap.interface.yaml | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/xyz/openbmc_project/Control/Power/Cap.interface.yaml b/xyz/openbmc_project/Control/Power/Cap.interface.yaml
index a293636..2994b14 100644
--- a/xyz/openbmc_project/Control/Power/Cap.interface.yaml
+++ b/xyz/openbmc_project/Control/Power/Cap.interface.yaml
@@ -13,15 +13,3 @@ properties:
       description: >
           Power cap enable.  Set to true to enable the PowerCap, false
           to disable it.
-
-    - name: CPUID1
-      type: string
-      description: >
-          CPU1 ID value.
-          The unit of measurement must be hexadecimal.
-
-    - name: CPUID2
-      type: string
-      description: >
-          CPU2 ID value.
-          The unit of measurement must be hexadecimal.
